CommonTools/RecoAlgos/python/sortedPFPrimaryVertices_cfi.py

```diff
@@ -22,7 +22,7 @@
     preferHighRanked = cms.bool(False),
 
     #options used in PUPPI
-    NumOfPUVtxsForCharged = cms.int32(-1),
+    NumOfPUVtxsForCharged = cms.uint32(0),
```

Comment:
good catch on the incorrect parameter type... it looks like `PFCandidatePrimaryVertexSorter` could also benefit from a fillDescriptions implementation?